src/VideoToolbox/VTProfessionalVideoWorkflow.cs

```diff
@@ -0,0 +1,36 @@
+// 
+// VTProfessionalVideoWorkflow.cs: VideoTools Professional Video Workflow 
+//
+// Authors: 
+//    Israel Soto (issoto@microsoft.com)
+// 
+// Copyright 2022 Microsoft Corporation.
+//
+
+#if __MACOS__
```

Comment:
```suggestion
#if MONOMAC
```
or are these synonymous?

Reply:
Yes, they are synonyms. Personally I prefer the underscore versions, because they match what works in normal iOS/tvOS/macOS projects (while `MONOMAC` is special to our own source code), but either works.